### ISSUE TITLE
Create tests for tabs / session management

### ISSUE THREAD

**Issue by sblatz** [mozilla-mobile/fenix#3667]

**Comment by boek** [comment]
Waiting on new home screen UI

**Comment by liuche** [comment]
Waiting on #3860, which might be put on hold, so removing waiting label because it's not in an upcoming sprint.

**Comment by boek** [comment]
This is fixed with #1063, UI and Unit tests!
